packages/web-components/fast-foundation/src/tree-item/tree-item.ts

```diff
@@ -102,9 +102,11 @@ export class TreeItem extends FoundationElement {
         }
     };
 
-    public handleClick = (e: MouseEvent): void => {
+    public handleClick = (e: MouseEvent): void | boolean => {
         if (!e.defaultPrevented) {
             this.handleSelected(e);
+            // do not prevent default as it completely eats the click
+            return true;
```

Comment:
does it eat the click or do we just not act appropriately because of the `if(!e.defaultPrevented)` check? why are we checking for defaultPrevented here?

Reply:
Removing the check here doesn't fix (I am honestly not sure why we are checking here...) - we're definitely eating the click and preventing default as part of our event handling.